app/services/settings/settings.ts

```diff
@@ -156,6 +156,10 @@ class SettingsViews extends ViewHandler<ISettingsServiceState> {
     return Utils.numberToBinnaryArray(this.values.Output.RecTracks, 6).reverse();
   }
 
+  get streamService() {
+    return this.values.Stream.service;
+  }
```

Comment:
i think we should maybe consider a different name here to avoid confusion since we have so many things named `xService`, i think `streamPlatform` would make sense here even if the property is called `service`